src/3d/shaders/mesh/mesh.geom

```diff
@@ -0,0 +1,160 @@
+#version 330 core
```

Comment:
it would be good to point to the original code of this file for reference

Reply:
sure!